src/pages/Trace/Instructions/Instructions.tsx

```diff
@@ -17,9 +17,13 @@ export const Instructions: React.FC<InstructionsProps> = ({curPathType, setPathT
 
```

Comment:
Add a todo: do not show the done button the first time we show instructions

Reply:
updated.